compiler/src/dotty/tools/dotc/transform/Flatten.scala

```diff
@@ -27,7 +27,20 @@ class Flatten extends MiniPhaseTransform with SymTransformer { thisTransform =>
     else ref
   }
 
-  private val liftedDefs = new mutable.ListBuffer[Tree]
+  private var liftedDefs = new mutable.ListBuffer[Tree]
+
+  private lazy val liftedDefsQueue = mutable.Stack.empty.asInstanceOf[mutable.Stack[mutable.ListBuffer[Tree]]]
```

Comment:
Also Stack is deprecated: https://www.scala-lang.org/api/current/scala/collection/mutable/Stack.html